CMakeLists.txt

```diff
@@ -106,6 +115,10 @@ set(COMMON_COMPILE_OPTIONS
     -Wundef \
     -Wunreachable-code \
     ")
+else ()
+    set(COMMON_COMPILE_OPTIONS
+    "${COMMON_COMPILE_OPTIONS} /wd4068 /wd4091 /wd4100 /wd4132 /wd4200 /wd4201 /wd4204 /wd4206 /wd4221 /wd4244 /wd4245")
```

Comment:
could you split these onto different lines and add comments as to what each one is for.

Reply:
Will do